NorthstarDedicatedTest/bansystem.cpp

```diff
@@ -38,8 +57,18 @@ void ServerBanSystem::ClearBanlist()
 	m_sBanlistStream.open(GetNorthstarPrefix() + "/banlist.txt", std::ofstream::out | std::ofstream::binary);
 }
 
+bool firstBanThisSession = true;
 void ServerBanSystem::BanUID(uint64_t uid)
 {
+	// if we do comments stuff etc theres a big chance that we fuck with the formatting and there wont be a newline at the end. if we then
+	// ban someone that uid will just be added directly after the last one, leading to invalid uids and confusion, so we just make a newline
+	// every time we ban someone for the first time
+	if (firstBanThisSession)
```

Comment:
should probably only do this if the last char isn't a newline

Reply:
did it hopefully the most efficient way